Rocket chat (#1229)
n8n-io/n8n-docs: docs/integrations/builtin/app-nodes/n8n-nodes-base.rocketchat.md

# Rocket.Chat

> The Rocket.Chat node allows you to automate work in Rocket.Chat, and integrate Rocket.Chat with other applications. n8n has built-in support for a wide range of Rocket.Chat features, including posting a message to a channel or a direct message. 

Reviewer: ```suggestion
The Rocket.Chat node allows you to automate work in Rocket.Chat, and integrate Rocket.Chat with other applications. n8n supports posting messages to channels, and sending direct messages, with Rocket.Chat. 
```
